### PR TITLE
fixes #45

### LINKED ISSUE #45: EntityAlreadyExistsException
This plugin worked the first time I tried it, but every time after that I get the following error:

> EntityAlreadyExistsException: An error occurred (EntityAlreadyExists) when calling the UploadServerCertificate operation. The Server Certificate with name xxxxxx already exists.

Should the plugin be able to replace the certificate automatically? Or is it expected that we manually delete old certificates?

### PR DESCRIPTION
**Pull request by michaelandrew**

### CODE REVIEW

**Comment by dlapiduz** [comment]
THanks @michaelandrew!

File: certbot_s3front/installer.py [hunk 1]
```diff
@@ -60,6 +60,21 @@ def deploy_cert(self, domain, cert_path, key_path, chain_path, fullchain_path):
...
+                ServerCertificateName=name
+            )
+        except botocore.exceptions.ClientError as e:
+            logger.error(e)
+
+        # Rename cert to the new one
+        client.update_server_certificate(
+            ServerCertificateName=name + '-new',
+            NewServerCertificateName=name
+        )
```
Comment (yungchin) [inline certbot_s3front/installer.py]: @dlapiduz I believe this fix needs rolling back: note that the rename now comes before the "-new" cert is even uploaded, which doesn't make sense to me.

See also my comment just now on #45 - I believe the problem we're trying to fix here was due to a bug in AWS, not in certbot-s3front.

Comment (dlapiduz) [inline certbot_s3front/installer.py]: Yes you are right, sorry about it.

Comment (yungchin) [inline certbot_s3front/installer.py]: Thanks for looking into it so promptly, and thanks for certbot-s3front!!

**Pull request merged**